lib/season_statistics.rb

```diff
@@ -0,0 +1,43 @@
+require 'pry'
+
+class SeasonStatistics 
+    def initialize(data)
+        @data = data
+    end
+    def season_game_id_verification 
```

Comment:
comparing first 4 digits of season to 4first for digits of game ID to eliminate the need to use teams.csv file